src/data_operations.py

```
@@ -0,0 +1,100 @@
+    data_df[col_date] = data_df[col_date].dt.date
+    return data_df
+
+def save_data(data_df, data_type, dataPath, SAVE_AS_PICKLE):
+    csv_dir = dataPath / data_type
+    csv_dir.mkdir(parents=True, exist_ok=True)
+    data_df.to_csv(csv_dir / "data.csv", index=False)
+    if SAVE_AS_PICKLE:
+        with open(csv_dir / f"{data_type}.pkl", 'wb') as f:
+            pickle.dump(data_df, f)
+
+def collect_audit_info(data_df, series_name, data_ref):
```

Ruff (0.8.2)

Missing return type annotation for public function `collect_audit_info`

(ANN201)